packages/kbn-es/src/serverless_resources/README.md

```diff
@@ -2,6 +2,17 @@
 
 The resources in this directory are used for seeding Elasticsearch Serverless images with users, roles and tokens for SSL and authentication. Serverless requires file realm authentication, so we will bind mount them into the containers at `/usr/share/elasticsearch/config/`.
 
+## Roles
+
+Roles defined in `roles.yml` is a combination of roles from `project-controller` like [security roles](https://github.com/elastic/project-controller/blob/main/internal/project/security/config/roles.yml).
```

Comment:
Can we remove this link and reverse the direction if necessary?  Generally we want to go from private repositories -> public repositories.  Instead we can say something like "roles defined here intended to mock a serverless deployment"

Reply:
Good point 👍 I've updated the README.

Reply:
Thank you!